packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -167,7 +168,7 @@ const DomainStep: React.FunctionComponent< DomainStepProps > = ( {
 							onExistingSubdomainSelect={ onExistingSubdomainSelect }
 							initialDomainSearch={ initialDomainSearch }
 							showSearchField={ false }
-							analyticsFlowId="focused-launch"
+							analyticsFlowId={ FOCUSED_LAUNCH_FLOW_ID }
 							analyticsUiAlgo="focused_launch_domain_picker"
```

Comment:
Changing this to something like `summary_domain_step` would be more aligned with https://github.com/Automattic/wp-calypso/pull/47318/files#diff-a88807ae898bfacf7689a16c723644ee8d5ddf40677fd0ce119761a404d0b897R26

Reply:
Thanks @razvanpapadopol, done.